blue-xdma/src/include/XDMADescriptorGenerator.bsv

```diff
@@ -0,0 +1,159 @@
+import FIFOF::*;
+import Clocks :: *;
+import GetPut :: *;
+import BlueAXI :: *;
+import BlueLib :: *;
+import AXI4LiteMaster :: *;
+import StmtFSM :: *;
+`include "Config.defines"
```

Comment:
do not use include, define Config.defines as BSV module and import it

Reply:
This has been changed to the method form in the latest commit.